### PR TITLE
Lint fixes with golangci-lints

### PR DESCRIPTION
**Pull request by Raviv-S**

### CODE REVIEW

File: manager/controllers/app/blueprint_controller.go [hunk 1]
```diff
@@ -336,8 +337,10 @@ func (r *BlueprintReconciler) reconcile(ctx context.Context, log zerolog.Logger,
 		if err != nil {
 			return ctrl.Result{}, errors.WithMessage(err, "Blueprint step arguments are invalid")
 		}
+
 		logging.LogStructure("Arguments", args, log, false, false)
-		releaseName := utils.GetReleaseName(blueprint.Labels[fapp.ApplicationNameLabel], blueprint.Labels[fapp.ApplicationNamespaceLabel], instanceName)
+    releaseName := utils.GetReleaseName(blueprint.Labels[fapp.ApplicationNameLabel],
```
Comment (shlomitk1) [inline manager/controllers/app/blueprint_controller.go:342]: Why is it shifted?

File: manager/controllers/app/fybrikapplication_conditions.go [hunk 2]
```diff
@@ -20,6 +20,7 @@ const (
 	DenyConditionIndex int64 = 1
 	// ErrorCondition means that an error was encountered during blueprint construction
 	ErrorConditionIndex int64 = 2
+	conditionsAmount    int   = 3
```
Comment (shlomitk1) [inline manager/controllers/app/fybrikapplication_conditions.go:23]: numConditions sounds better

Comment (Raviv-S) [reply manager/controllers/app/fybrikapplication_conditions.go]: Thanks, fixed

File: manager/controllers/app/blueprint_controller.go (hunk 1 above)
Comment (Raviv-S) [reply manager/controllers/app/blueprint_controller.go]: Fixed

File: manager/controllers/app/fybrikapplication_controller.go [hunk 3]
```diff
@@ -666,7 +682,9 @@ func (r *FybrikApplicationReconciler) updateProvisionedStorageStatus(application
 	return true, nil
 }
 
-func (r *FybrikApplicationReconciler) buildSolution(applicationContext ApplicationContext, requirements []DataInfo) (map[string]NewAssetInfo, *api.PlotterSpec, error) {
+//nolint:
```
Comment (shlomitk1) [inline manager/controllers/app/fybrikapplication_controller.go:685]: what does an empty statement mean?

File: manager/controllers/app/plotter_controller.go [hunk 4]
```diff
@@ -308,6 +308,7 @@ func (r *PlotterReconciler) setPlotterAssetsReadyStateToFalse(assetToStatusMap m
 	}
 }
 
+//nolint:funlen,gocyclo
```
Comment (shlomitk1) [inline manager/controllers/app/plotter_controller.go:311]: For this and other //nolint comments: can you create an issue with a list of all linting exceptions that were made? Perhaps we can do some refactoring according to the found lint issues.

File: manager/controllers/app/fybrikapplication_controller.go (hunk 3 above)
Comment (Raviv-S) [reply manager/controllers/app/fybrikapplication_controller.go:685]: A mistake, I'll delete that line

File: manager/controllers/app/plotter_controller.go (hunk 4 above)
Comment (Raviv-S) [reply manager/controllers/app/plotter_controller.go:311]: Sure.
FYI: For some of the specific problems on code performance I don't add `//nolint` so it'll be easier to spot.

Comment (roytman) [reply manager/controllers/app/plotter_controller.go:311]: hi @shlomitk1 , we plan to run additional lint fixes.

**Pull request merged**